src/utils/avatar.js

```diff
@@ -284,7 +286,18 @@ export class FallbackAvatarURL extends AvatarURL {
       this._standardUrl = new URL(this._standardUrl);
     }
 
-    return this._standardUrl;
+    let result: URL = this._standardUrl;
+    if (sizePhysicalPx > 100) {
```

Comment:
Can you leave a comment explaining where `100` comes from?

Reply:
Ah, good point—this was copied from `UploadedAvatarURL#get`, which grew around some lines first added in aec2ff255, I think, without an implementation comment there—and I never added one.

Reply:
Er, rather, I addressed this in a commit that comes after this one; hopefully that's fine. So, there was an existing lack (the one in `UploadedAvatarURL`), I propagated it just a bit, here, then fixed both.